src/modules/blog/controllers/blog.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Post, Body, HttpException, HttpStatus, UseGuards } from '@nestjs/common';
+import { BlogService } from '../services/blog.service';
+import { CreateBlogDto } from '../dto/create-blog.dto';
+import { Blog } from '../entities/blog.entity';
+import { AuthGuard } from '../../../guards/auth.guard';
+@Controller('blogs')
+export class BlogController {
+  constructor(private readonly blogService: BlogService) {}
+
+  @UseGuards(AuthGuard)
+  @Post()
+  async create(
```

Comment:
Document your routes, and remove @UseGuards() because the guard is available globally